web/pyproject.toml

```diff
@@ -60,3 +60,9 @@ ignore_missing_model_attributes = true
 
 [tool.doc8]
 max_line_length = 110
+
+[tool.isort]
+multi_line_output = 3
+skip = "migrations"
+line_length = 88
+include_trailing_comma = true
```

Comment:
lets add profile settings which will be compatible with black. Line lenth 110

Reply:
Got it

Reply:
resolved this issue in c9ec021d3d35b28b254470662efc97b210c7035f commit